workspace/utilities/com/schema-organization.xsl

```diff
@@ -2,27 +2,39 @@
 <xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
 
 	<xsl:template name="schema-organization">
-		<xsl:param name="address" select="''" />
-		<xsl:param name="region" select="''" />
-		<xsl:param name="country" select="''" />
-		<xsl:param name="postal-code" select="''" />
-		<xsl:param name="phone" select="''" />
-		<xsl:param name="email" select="''" />
-		<xsl:param name="name" select="''" />
+		<xsl:param name="phone" select="$config/organization-phone" />
+		<xsl:param name="email" select="$config/organization-email" />
+		<xsl:param name="name" select="$config/organization-name" />
+		<xsl:param name="logo" select="$config/organization-logo" />
+		<xsl:param name="logo-path">
+			<xsl:if test="string-length($logo) != 0">
+				<xsl:if test="string-length($logo/@path) != 0 and string-length($logo/filename) != 0">
+					<xsl:value-of select="$root" />
+					<xsl:text>/workspace</xsl:text>
+					<xsl:value-of select="$logo/@path" />
+					<xsl:text>/</xsl:text>
+					<xsl:value-of select="$logo/filename" />
+				</xsl:if>
+			</xsl:if>
+		</xsl:param>
 
 		<script type="application/ld+json">
 		{
 			"@context": "http://schema.org",
 			"@type": "Organization",
-			"address": {
-			"@type": "PostalAddress",
-			"addressLocality": "<xsl:value-of select="translate(concat($region, ' ',$country), ',&quot;', '')" />",
-			"postalCode": "<xsl:value-of select="translate($postal-code, ',&quot;', '')" />",
-			"streetAddress": "<xsl:value-of select="translate($address, ',&quot;', '')" />"
-			},
```

Comment:
Why remove address ?

Reply:
I'd rather split Organization and LocalBusiness.
Not all our customers have "Pignon sur rue". And if they have, i can be waaaay more specific with localbusiness
https://developers.google.com/search/docs/data-types/local-business

Reply:
Yeah, but I would make the address available (and optional).

That address can be the HQ, that can differ from LocalBusiness... no ?